face_track/main.py

```diff
@@ -27,9 +27,9 @@
 
 print "Start face-tracking node"
 ft = FaceTrack()
-
+ft.do_look_at_actions()
 while not rospy.is_shutdown():
-	ft.do_look_at_actions()
+	#ft.do_look_at_actions()
```

Comment:
I assume that `rospy.spin()` is probably a call to unix `select()` under the covers, instead of an actual busy-spin loop.  select is always more efficient, it lets the OS kernel schedule stuff.

The problem here is that while you are sleeping for 1/10th of a second, ros cannot really respond to anything. This was OK while we kept things simple, but now that you're getting fancy, its not the right way to do things.